portfolio/src/main/webapp/index.html

```diff
@@ -100,8 +110,8 @@ <h1>Comments</h1>
       </div>
       <div class="col-6 center-text margin-bottom">
         <form action="/data" method="POST">
-          <textarea name="name-input" class="form-control" id="exampleFormControlTextarea1" rows="1">Name</textarea>				  
-          <textarea name="text-input" class="form-control" id="exampleFormControlTextarea1" rows="3">Comment...</textarea>
+          <textarea name="name" class="form-control" id="exampleFormControlTextarea1" rows="1">Name</textarea>				  
```

Comment:
Nit: It looks like you have 2 textareas with the same id, you might want to change the ids for each of these to be more descriptive and unique, otherwise you might run into issues when referencing these in JavaScript or CSS

Reply:
Okay, thanks. This is fixed in the next PR where I end up needing to reference them in JavaScript